CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if(POLICY CMP0051)
   cmake_policy(SET CMP0051 NEW)
 endif()
 
+if(POLICY CMP0054)
```

Comment:
That looks correct to me. CMP0054 controls `if("CMAKE_SYSTEM_NAME" STREQUAL ...`.